plugin_runner/plugin_runner.py

```diff
@@ -62,7 +60,8 @@ async def HandleEvent(self, request: Event, context):
             protocol_class = plugin["class"]
 
             try:
-                effects = protocol_class(request, plugin.get("secrets", {})).compute()
+                protocol = protocol_class(request, plugin.get("secrets", {}))
+                effects = await asyncio.get_running_loop().run_in_executor(None, protocol.compute)
```

Comment:
This uses default executor settings, which means a threadpool with the default number of max workers. From the Python docs:

> Changed in version 3.8: Default value of max_workers is changed to min(32, os.cpu_count() + 4). This default value preserves at least 5 workers for I/O bound tasks. It utilizes at most 32 CPU cores for CPU bound tasks which release the GIL. And it avoids using very large resources implicitly on many-core machines.